src/workerd/api/node/tests/crypto_sign-test.js

```diff
@@ -144,3 +145,37 @@ export const dsaSignVerifyObjects = {
     });
   },
 };
+
+export const testSignLength = {
+  test() {
+    // Tests that generated signatures are not overly long.
+    const message = `Test message 123: ${Math.random().toString(36).substring(2, 15)}`;
+
+    const keyPair = generateKeyPairSync('ec', {
+      namedCurve: 'prime256v1',
+      publicKeyEncoding: {
+        type: 'spki',
+        format: 'pem',
+      },
+      privateKeyEncoding: {
+        type: 'pkcs8',
+        format: 'pem',
+      },
+    });
+
+    for (let n = 0; n < 1000; n++) {
```

Comment:
isn't this testing 1000 times the same thing?
_(should `const message =...` be moved inside the loop)_